test/e2e/package.json

```diff
@@ -10,6 +10,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "lodash": "^3.9.3",
-    "semver": "^5.3.0"
+    "semver": "^5.3.0",
+    "thenify-all": "^1.6.0"
```

Comment:
@google-cloud/common has a promisifyAll function you could use and avoid this dependency.

https://github.com/GoogleCloudPlatform/google-cloud-node/blob/master/packages/common/src/util.js#L743